src/licensedcode/data/rules/mit_and_apache-2.0_10.RULE

```diff
@@ -0,0 +1,13 @@
+---
+license_expression: mit AND apache-2.0
```

Comment:
```suggestion
license_expression: mit OR apache-2.0
```

As weird as it sounds this is choice of license, not a conjunction. This is the license of BUCK afaik at https://github.com/facebook/buck2 ans https://github.com/facebook/buck2/blob/732db19980a934de534193ec2d4e330a1d251575/ci.bzl#L3 and this a choice of licenses.